Remotion/Data/DomainObjects.UnitTests/Persistence/MixedStorageProviderTest.cs

```diff
@@ -30,7 +31,7 @@ public class MixedStorageProviderTest : StandardMappingTest
     [Test]
     public void StorageProvidersCanBeMixed ()
     {
-      using (MixinConfiguration.BuildFromActive().ForClass(typeof(StorageProvider)).Clear().AddMixins(typeof(StorageProviderWithFixedGuidMixin)).EnterScope())
+      using (MixinConfiguration.BuildFromActive().ForClass(typeof(IStorageProvider)).Clear().AddMixins(typeof(StorageProviderWithFixedGuidMixin)).EnterScope())
```

Comment:
can we mix interfaces or should this RdbmsProvider?

Reply:
Interfaces are valid mixin targets, so this works - all tests are green :-)

Reply:
/hides in fear/